NAMESPACE

```diff
@@ -5,14 +5,17 @@ export(
     "cfunction",
     "cxxfunction",
     "getPlugin",
+    "readCFunc",
     "registerPlugin",
-    "rcpp"
+    "rcpp",
+    "writeCFunc"
```

Comment:
Do these actually persist C-level functions across sessions now?  We were never clear if that could be done.

Reply:
Yes, that's exactly the point. I just ran

```
x <- as.numeric(1:10)
n <- as.integer(10)

code <- "
      integer i
      do 1 i=1, n(1)
    1 x(i) = x(i)**3
"
cubefn <- cfunction(signature(n="integer", x="numeric"), code,
  convention=".Fortran")
code(cubefn)

cubefn(n, x)$x

moveDLL(cubefn, name = "cubefn", directory = "~/dll")
path <- file.path("~/dll", "cubefn.rda")
writeCFunc(cubefn, path)
```

in one session and

```
path <- file.path("~/dll", "cubefn.rda")
cfn <- readCFunc(path)
cfn(3, 1:3)$x
```

and got

```
> cfn(3, 1:3)$x
[1]  1  8 27
```

using 0.3.16.2 on Linux

`\o/`